Source/WebCore/Modules/websockets/ThreadableWebSocketChannelClientWrapper.h

```diff
@@ -90,7 +90,7 @@ class ThreadableWebSocketChannelClientWrapper : public ThreadSafeRefCounted<Thre
 
     void processPendingTasks();
 
-    ScriptExecutionContext& m_context;
+    Ref<ScriptExecutionContext> m_context;
```

Comment:
These references probably need to be WeakPtr, not RefPtr.

Reply:
I'm not sure if that is correct - it currently holds a raw reference, so it's assuming the object to be alive? (so the changes should probably make that assumption true?)

Reply:
The previous code didn't look safe to me since ThreadableWebSocketChannelClientWrapper is ref counted. Someone could totally keep it alive longer than the Document.

Also note that switching a `T&` to a `Ref<T>&` is not always safe since it can introduce reference cycles and thus leaks.

Reply:
Ok, is the suggestion to change to WeakPtr for only this, or everywhere in this change?